api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/EligibilityOverrideTestDataFactory.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.dhsc.htbhf.claimant.testsupport;
+
+import uk.gov.dhsc.htbhf.claimant.entity.EligibilityOverride;
+import uk.gov.dhsc.htbhf.dwp.model.EligibilityOutcome;
+
+public class EligibilityOverrideTestDataFactory {
+
+    public static EligibilityOverride eligibilityConfirmedWithNoChildrenForFiveYears() {
```

Comment:
I don't see what FiveYears has to do with this object? Making a guess, it could either be how long the override applies for or the age of children, but its unclear. Either way, has nothing to do with the method, so I'd rename it to something simpler like `aConfirmedEligibilityOverride`.